### ISSUE TITLE
Crashes with python-pillow>4.1.1-1

### ISSUE THREAD

**Issue by th0rn** [amietn/vcsi#43]
Running vcsi with either default params (deja vu fonts are installed) or with setting fonts manually results in this error:

```
Composing contact sheet...
Traceback (most recent call last):
  File "/usr/bin/vcsi", line 11, in <module>
    load_entry_point('vcsi==6', 'console_scripts', 'vcsi')()
  File "/usr/lib/python3.6/site-packages/vcsi/__init__.py", line 3, in main
    vcsi.vcsi.main()
  File "/usr/lib/python3.6/site-packages/vcsi/vcsi.py", line 1202, in main
    process_file(path, args)
  File "/usr/lib/python3.6/site-packages/vcsi/vcsi.py", line 1252, in process_file
    image = compose_contact_sheet(media_info, selected_frames, args)
  File "/usr/lib/python3.6/site-packages/vcsi/vcsi.py", line 733, in compose_contact_sheet
    template_path=args.metadata_template_path)
  File "/usr/lib/python3.6/site-packages/vcsi/vcsi.py", line 669, in prepare_metadata_text_lines
    text=remaining_chars)
  File "/usr/lib/python3.6/site-packages/vcsi/vcsi.py", line 633, in max_line_length
    text_width = metadata_font.getsize(text_chunk)[0]
  File "/usr/lib/python3.6/site-packages/PIL/ImageFont.py", line 156, in getsize
    size, offset = self.font.getsize(text, direction, features)
SystemError: <built-in method getsize of Font object at 0x7f986eead990> returned NULL without setting an error
```

This is true both for when using python-pillow 4.2.1-1 and 4.2.0-1. No such error happens when using python-pillow 4.1.1-1. Using python 3.6.1 on Arch Linux 4.9.37-1-lts.

**Comment by amietn** [comment]
@th0rn See https://github.com/python-pillow/Pillow/issues/2614

I'll probably add a fix to not try and compute the size of empty strings in vcsi as well.

**Comment by amietn** [comment]
Working around other libraries' bugs feels wrong but here's a patch until python-pillow releases a new version that includes https://github.com/python-pillow/Pillow/pull/2624/commits/3dd3a906465dd7afc36c483b6dfd7554596618ea

**Comment by hugovk** [comment]
FYI: https://github.com/python-pillow/Pillow/issues/2614 has been fixed in this week's Pillow 4.3.0 release.
